skeleton/src/main/java/com/dropbox/kaiken/skeleton/skeleton/core/KaikenConfig.kt

```diff
@@ -0,0 +1,27 @@
+package com.dropbox.kaiken.skeleton.skeleton.core
+
+import com.dropbox.kaiken.skeleton.skeleton.dagger.SdkSpec
+import com.dropbox.kaiken.scoping.AppComponent
+import com.dropbox.kaiken.scoping.AppServices
+import com.dropbox.kaiken.scoping.SkeletonScope
+import com.dropbox.kaiken.scoping.UserComponent
+import com.dropbox.kaiken.scoping.UserParentComponent
+import com.dropbox.kaiken.scoping.cast
+import com.squareup.anvil.annotations.ContributesBinding
+import javax.inject.Inject
+
+@ContributesBinding(SkeletonScope::class)
+class KaikenConfig @Inject constructor(val app: SkeletonApplication) : SkeletonConfig {
```

Comment:
I made skeleton config injectable. That will allow us to provide specific skeleton configs to specific apps